ports-of-call/variant/lib.hpp

```diff
@@ -0,0 +1,493 @@
+// PortsOfCall::variant
+//
+// Copyright Michael Park, 2015-2017
+//
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE.md or copy at http://boost.org/LICENSE_1_0.txt)
+
+// GPU compatibility corrections are © 2025. Triad National Security,
+// LLC under U.S. Government contract 89233218CNA000001 for Los Alamos
+// National Laboratory (LANL), which is operated by Triad National
+// Security, LLC for the U.S. Department of Energy/National Nuclear
+// Security Administration.
+
+#ifndef _PORTS_OF_CALL_VARIANT_LIB_HPP_
+#define _PORTS_OF_CALL_VARIANT_LIB_HPP_
+
+#include <functional>
+#include <memory>
+#include <type_traits>
+#include <utility>
+
+#include <ports-of-call/portable_config.hpp>
+
+#define PORTABLE_RETURN(...)                                                             \
+  noexcept(noexcept(__VA_ARGS__))->decltype(__VA_ARGS__) { return __VA_ARGS__; }
+
+namespace PortsOfCall {
+namespace lib {
```

Comment:
similarly, this should be `PortsOfCall::variant::lib`

Reply:
it can't be `variant::lib` but it can be `Variant::lib` because `PortsOfCall::variant` is the class

Reply:
Or it could be `variant_lib`